PukingPredator/Assets/Scripts/Inventory.cs

```diff
@@ -206,6 +207,15 @@ public void UpdateUI()
             itemUIComponent.item = item;
         }
 
+
         inventoryUI.UpdateText(items.Count, maxCount);
     }
+
+    /// <summary>
+    /// Get the mass of the player, currently implemented as inventory item count
+    /// </summary>
+    public float getMass()
```

Comment:
Incorrect naming scheme for methods in the project, it would be "GetMass", and really that makes more sense as a computed property, which already exists ("totalMass"). This also isn't actually what it does though, so the name is misleading. It isn't getting the mass at all, it's getting the count. Just directly fetch inventory.itemCount if we're just handling it as your inventory having 5 items, don't hide that design behind a misleading name.